playbooks/cloud-provider-openstack-multinode-csi-migration-test/pre.yaml

```diff
@@ -18,14 +20,11 @@
           apiServer:
             extraArgs:
               "feature-gates": "CSIMigration=true,CSIMigrationOpenStack=true,ExpandCSIVolumes=true"
+              "cloud-provider": "external"
           controllerManager:
             extraArgs:
               "feature-gates": "CSIMigration=true,CSIMigrationOpenStack=true,ExpandCSIVolumes=true"
-          ---
-          apiVersion: kubeadm.k8s.io/v1beta2
-          kind: InitConfiguration
-          nodeRegistration:
-              name: {{ inventory_hostname }}
+              "cloud-provider": "external"
```

Comment:
@chrigl im confused, if we add cloud-provider as external, we wont be using intree anymore?

Reply:
In v1.18.0 we have new tests in the Kubernetes e2e suite. These ones now fail, because the test-cluster does not have proper topology labels assigned to the nodes. These could be either done by the old in-tree cloud-provider, or OCCM, or manually by hand. I decided to use OCCM, because the in-tree one is going away in the future, but more important, I don't want to pass in OS credentials into kube-controller-manager and kubelet. So we are really sure, the CSIMigration is used.

Reply:
got it. Thanks :)